docs/elements.md

```diff
@@ -814,9 +814,11 @@ def generate(data):
 | `allow-blank`         | boolean                         | false    | Whether or not an empty input box is allowed. By default, empty input boxes will not be graded (invalid format).                                                                                                                                                                                                                                                                                                                                                                        |
+| `show-score`          | boolean                         | true     | Whether to show the score badge next to this element.                                                                                                                                                                                                                                                                                                                                                                                                                                   |
 | `blank-value`         | string                          | 0 (zero) | Value to be used as an answer if element is left blank. Only applied if `allow-blank` is `true`. Must follow the same format as an expected user input (e.g., fractions if allowed, complex numbers if allowed, etc.).                                                                                                                                                                                                                                                                  |
 | `show-help-text`      | boolean                         | true     | Show the question mark at the end of the input displaying required input parameters.                                                                                                                                                                                                                                                                                                                                                                                                    |
 | `show-placeholder`    | boolean                         | true     | Show the placeholder text that shows the default comparison.                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
The use of `show-placeholder` is deprecated with the introduction of the custom `placeholder` attribute. It should be removed from the docs (but should stay in the code, the current implementation is correct).

Reply:
so ill just remove the `show-placeholder` line then?

Reply:
> so ill just remove the `show-placeholder` line then?

Yep!

Reply:
Fixed in [616c262](https://github.com/PrairieLearn/PrairieLearn/pull/7934/commits/616c26287a9ad0f2d323ed38ffe3aa742b5a2bcd).